_studio/shared/umc/codec/h264_dec/src/umc_h264_task_supplier.cpp

```diff
@@ -1186,13 +1183,13 @@ void MVC_Extension::ChooseLevelIdc(const H264SeqParamSetMVCExtension * extension
     if (m_level_idc)
         return;
 
-    VM_ASSERT(extension->viewInfo.Size() == extension->num_views_minus1 + 1);
+    assert(extension->viewInfo.size() == extension->num_views_minus1 + 1);
 
-    if (!m_viewIDsList.size())
+    if (m_viewIDsList.empty())
     {
-        for (size_t i = 0; i <= extension->num_views_minus1; i++)
+        for (const auto & viewInfo : extension->viewInfo)
```

Comment:
before the for loop add small optimization:
```
m_viewIDsList.reserve(extension->viewInfo.size());
```

Reply:
std::list<T> hasn't member reserve